articles/quickstart/webapp/nodejs/02-user-profile.md

```diff
@@ -16,11 +16,10 @@ budicon: 292
 
 ## User Profile
 
-Getting the user's profile information is very simple with Auth0. After the user has authenticated, a `user` object
-with the entire profile is attached to every express request. By default this will be a minimal profile, to retrieve additional profile information
-the `profile` scope should be added to the authentication scopes.
+After the users has authenticated, every express request has a `user` object with the entire user profile. 
```

Comment:
`profile` does not help much (assuming that we are talking about oidc-conformant clients).

Even that will return only a few claims, see here https://auth0.com/docs/scopes/current#example-asking-for-standard-claims

If the user needs more user info they can use the Management API https://auth0.com/docs/api/management/v2#!/Users/get_users_by_id which returns sth like that:

```
{
  "email": "john.doe@gmail.com",
  "email_verified": false,
  "username": "johndoe",
  "phone_number": "+199999999999999",
  "phone_verified": false,
  "user_id": "usr_5457edea1b8f33391a000004",
  "created_at": "",
  "updated_at": "",
  "identities": [
    {
      "connection": "Initial-Connection",
      "user_id": "5457edea1b8f22891a000004",
      "provider": "auth0",
      "isSocial": false
    }
  ],
  "app_metadata": {},
  "user_metadata": {},
  "picture": "",
  "name": "",
  "nickname": "",
  "multifactor": [
    ""
  ],
  "last_ip": "",
  "last_login": "",
  "logins_count": 0,
  "blocked": false,
  "given_name": "",
  "family_name": ""
}
```